numpy/distutils/fujitsuccompiler.py

```diff
@@ -0,0 +1,28 @@
+from distutils.unixccompiler import UnixCCompiler
+
+class FujitsuCCompiler(UnixCCompiler):
+
+    """
+    Fujitsu compiler.
+    """
+
+    compiler_type = 'fujitsu'
+    cc_exe = 'fcc'
+    cxx_exe = 'FCC'
+
+    def __init__(self, verbose=0, dry_run=0, force=0):
+        UnixCCompiler.__init__(self, verbose, dry_run, force)
+        cc_compiler = self.cc_exe
+        cxx_compiler = self.cxx_exe
+        self.set_executables(
+            compiler=cc_compiler +
+            ' -O3 -Nclang -fPIC -fno-vectorize -march=armv8.2-a+nosve',
+            compiler_so=cc_compiler +
+            ' -O3 -Nclang -fPIC -fno-vectorize -march=armv8.2-a+nosve',
+            compiler_cxx=cxx_compiler +
+            ' -O3 -Nclang -fPIC -fno-vectorize -march=armv8.2-a+nosve',
+            linker_exe=cc_compiler +
+            ' -O3 -Nclang -fPIC -lfj90i -lfj90f -lfjsrcinfo -lelf -shared',
+            linker_so=cc_compiler +
+            ' -O3 -Nclang -lfj90i -lfj90f -lfjsrcinfo -lelf -shared'
```

Comment:
Adding compiler options here seems not correct? `-O3` is not necessarily best for NumPy anyway, although I suppose it may be for many use-cases (at least when not on Linux, since we are probably a bit sloppy with the `NPY_GCC_OPT_3` macro, which *could* be extended!)

Reply:
You are right, there is no need to specify `-O3` in the linker options. 
Sorry, I overlooked that. I will correct it.